packages/new-design/lib/utils.mjs

```diff
@@ -253,29 +264,44 @@ const showTips = (config, choices) => {
 // Creates the environment based on the user's choices
 export const createEnvironment = async (choices) => {
   // Store directories for re-use
-  ;(config.cwd = cwd),
-    (config.source = {
-      root: cwd,
-      template: cwd + `/../templates/from-${choices.template}`,
-      shared: cwd + `/../shared`,
-    })
+  config.source = {
+    template: `${newDesignDir}/templates/from-${choices.template}`,
+    shared: `${newDesignDir}/shared`,
+  }
   config.dest = join(process.cwd(), choices.name)
 
   // Create target directory
   await mkdir(config.dest, { recursive: true })
 
   // Find files
-  config.files = {
-    template: await rdir(config.source.template),
-    shared: await rdir(config.source.shared),
+  config.relativeFiles = {
+    template: (await rdir(config.source.template)).map((file) =>
+      relative(config.source.template, file)
+    ),
+    shared: (await rdir(config.source.shared)).map((file) => relative(config.source.shared, file)),
   }
 
   // Output a linebreak
   console.log()
 
   // Copy/Template files
   try {
-    await oraPromise(copyTemplate(config, choices), {
+    const templateVars = {
+      Name: capitalize(choices.name),
+      name: choices.name,
+      tag: config.tag,
+    }
+
+    try {
+      await stat(join(monorepoDesignsDir, choices.template))
```

Comment:
@nchilada can you say more about the purpose of this? Is it a check to make sure that the intended inherited design exists so that the block argument can be set?